buildenv

```diff
@@ -1,8 +1,13 @@
-export ZOPEN_GIT_URL="https://github.com/justjanne/powerline-go.git"
-export ZOPEN_GIT_DEPS="comp_go"
-export ZOPEN_TYPE="GIT"
+# bump: powerline-go-version /POWERLINE_VERSION="(.*)"/ https://github.com/justjanne/powerline-go.git|semver:*
+POWERLINE_VERSION="1.24"
+
+export ZOPEN_BUILD_LINE="STABLE"
+export ZOPEN_STABLE_DEPS="comp_go"
+export ZOPEN_STABLE_URL="https://github.com/justjanne/powerline-go/archive/refs/tags/v${POWERLINE_VERSION}.tar.gz"
```

Comment:
Can we change this use ZOPEN_STABLE_TAG?
```suggestion
export ZOPEN_STABLE_TAG="v${POWERLINE_VERSION}"
```

Reply:
I see that it makes the URL clean-looking as one benefit, and also that the pulled repo will then be git-initialized.
But is the latter required, considering that the updates are upstreamed already? 
I know there is one patch in the folder but I doubt if it'll get upstreamed.

I'm not clear on where to use `.tar.gz` vs `.git` and `STABLE_TAG`, as there's a mix of both across the repos.